e2e/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [
+    "@integreat-app/eslint-config-jest",
+    "@integreat-app/eslint-config-flow",
+    "@integreat-app/eslint-config-react",
```

Comment:
react is not needed for e2e tests. If there is react code then the test is wrong.